pkg/kubelet/cadvisor.go

```diff
@@ -59,7 +59,7 @@ func (kl *Kubelet) GetContainerInfo(podFullName string, uid types.UID, container
 	if cc == nil {
 		return nil, nil
 	}
-	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.dockerClient, false)
+	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.containerRuntime, false)
```

Comment:
For now, I think it's fine to assume that all Kubelets are configured with the same runtime. Otherwise, we'd need to surface information about which runtime was supported by each node so that the scheduler could choose nodes able to run the images specified by the pods' containers.